<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nCompareLines</class>
 <widget class="QMainWindow" name="nCompareLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="nCustomPlot" name="plot" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="image">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="neutrinoImage" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addImage">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Add image lineout</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../../private/var/folders/0v/r34b88ss03z9spv7tk_cs0nr0000gp/T/qtc-cmake-cDUc1L/src/qrc/icons.qrc">
           <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeImage">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Remove image lineout</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../../private/var/folders/0v/r34b88ss03z9spv7tk_cs0nr0000gp/T/qtc-cmake-cDUc1L/src/qrc/icons.qrc">
           <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="current">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Show current image lineout</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSaveClipboard"/>
   <addaction name="actionSaveTxt"/>
   <addaction name="actionSavePDF"/>
   <addaction name="separator"/>
   <addaction name="actionAddAll"/>
   <addaction name="actionRemoveAll"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionSaveClipboard">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveClipboard.png</normaloff>:icons//saveClipboard.png</iconset>
   </property>
   <property name="text">
    <string>saveClipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionSaveTxt">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveTxt.png</normaloff>:icons//saveTxt.png</iconset>
   </property>
   <property name="text">
    <string>saveTxt</string>
   </property>
   <property name="toolTip">
    <string>Export TXT</string>
   </property>
  </action>
  <action name="actionSavePDF">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePDF.png</normaloff>:icons//savePDF.png</iconset>
   </property>
   <property name="text">
    <string>savePDF</string>
   </property>
   <property name="toolTip">
    <string>Export PDF or SVG</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset>
     <normaloff>:icons//line.png</normaloff>:icons//line.png</iconset>
   </property>
   <property name="text">
    <string>rect</string>
   </property>
   <property name="toolTip">
    <string>Show Region</string>
   </property>
  </action>
  <action name="actionAddAll">
   <property name="icon">
    <iconset resource="../../../../../../private/var/folders/0v/r34b88ss03z9spv7tk_cs0nr0000gp/T/qtc-cmake-cDUc1L/src/qrc/icons.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>addAll</string>
   </property>
   <property name="toolTip">
    <string>add all images</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="icon">
    <iconset resource="../../../../../../private/var/folders/0v/r34b88ss03z9spv7tk_cs0nr0000gp/T/qtc-cmake-cDUc1L/src/qrc/icons.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>removeAll</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nCustomPlot</class>
   <extends>QWidget</extends>
   <header>nCustomPlots.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../private/var/folders/0v/r34b88ss03z9spv7tk_cs0nr0000gp/T/qtc-cmake-cDUc1L/src/qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
